doc/source/install.rst

```diff
@@ -15,6 +15,23 @@ To install the `satpy` package and all of its python dependencies from PyPI:
 
     $ pip install satpy
 
+To install with optional packages:
+
+.. code-block:: bash
+
+    $ pip install satpy pyorbital pyproj pycoast pyspectral pygac aggdraw h5netcdf python-hdf4 dask xarray toolz numexpr
+
+Example for Ubuntu using virtualenv:
+
+.. code-block:: bash
+
+    $ sudo apt-get install python-pip python-gdal
+    $ sudo pip install virtualenv
```

Comment:
I'm not a huge fan of suggesting that users install stuff as `sudo`. I know it is necessary if you are using the system python, but I'm wondering if there is a better way. I would say conda, but I know not everyone uses conda.